gate-web/config/gate.yml

```diff
@@ -81,13 +81,35 @@ spring:
       SORT_PROPERTIES_ALPHABETICALLY: true
     serialization:
       ORDER_MAP_ENTRIES_BY_KEYS: true
-#  oauth2:
-#      client:
-#        clientId:
-#        clientSecret:
-#        accessTokenUri:
-#        userAuthorizationUri:
-#        scope:
-#          - email
-#      resource:
-#        userInfoUri:
+
+---
+
+spring:
+  profiles: googleOAuth
+  oauth2:
+    client:
+      # Override these values in your own gate-googleOAuth.yml file
+      # clientId:
+      # clientSecret:
+      accessTokenUri: https://www.googleapis.com/oauth2/v4/token
+      userAuthorizationUri: https://accounts.google.com/o/oauth2/v2/auth?access_type=offline
+      scope: "profile email"
+    resource:
+      userInfoUri: https://www.googleapis.com/oauth2/v3/userinfo
+
+
+---
+
+spring:
+  profiles: azureOAuth
+  oauth2:
+    client:
+      # Override these values in your own gate-azureOAuth.yml file
+      # clientId:
+      # clientSecret:
+      # accessTokenUri: https://login.microsoftonline.com/<tenant_id>/oauth2/token
```

Comment:
@scotmoor - Scot, are you good with the names of config values?  Agree, user should override in spinnaker-local.yml.  @scotmoor could we use values clientId, clientSecret, tenantId and form the accessTokenUri in code instead?